workshop/Lab1/README.md

```diff
@@ -11,69 +11,85 @@ If you haven't already:
-the "guestbook" application.
+Once your client is configured, you are ready to deploy your first application, `guestbook`.
 
-# 2. Deploy your application
+# 1. Deploy your application
 
 In this part of the lab we will deploy an application called `guestbook`
 that has already been built and uploaded to DockerHub under the name
 `ibmcom/guestbook:v1`.
 
-1. Run `$ bx cs cluster-config <name-of-cluster>`, and set the `KUBECONFIG`
-   environment variable based on the output of the command. This will
-   make your `kubectl` client point to your new Kubernetes cluster.
-
-2. Start by running `guestbook` as a deployment:
+1. Start by running `guestbook`:
 
    ```$ kubectl run guestbook --image=ibmcom/guestbook:v1```
 
-   This action will take a bit of time. To check the status of your deployment,
+   This action will take a bit of time. To check the status of the running application,
    you can use `$ kubectl get pods`.
 
    You should see output similar to the following:
 
-   ```
+   ```console
    $ kubectl get pods
    NAME                          READY     STATUS              RESTARTS   AGE
    guestbook-59bd679fdc-bxdg7    0/1       ContainerCreating   0          1m
    ```
-
+   Eventually, the status should show up as `Running`.
+   
+   ```console
+   $ kubectl get pods
+   NAME                          READY     STATUS              RESTARTS   AGE
+   guestbook-59bd679fdc-bxdg7    1/1       Running             0          1m
+   ```
+   
+   The end result of our the run command is not just the pod containing our application containers,
+   but a Deployment resource that manages the lifecycle of those pods.
+   We can look at the details of the deployment with `kubectl describe deployment guestbook`.   
+   
 3. Once the status reads `Running`, we need to expose that deployment as a
    service so we can access it through the IP of the worker nodes.
-   The example for this course listens on port 8080.  Run:
+   The `guestbook` application listens on port 3000.  Run:
 
-   ```$ kubectl expose deployment/guestbook --type="NodePort" --port=3000```
+   ```console
+   $ kubectl expose deployment guestbook --type="NodePort" --port=3000
+   service "guestbook" exposed
+   ```
 
 4. To find the port used on that worker node, examine your new service:
 
-   ```$ kubectl describe service guestbook```
-
-   Take note of the "NodePort:" line as `<nodeport>`
-
-5. Run `$ bx cs workers <name-of-cluster>`, and note the public IP listed
-   on the `<public-IP>` line.
-
-6. You can now access the application using
-   `$ curl <public-IP>:<nodeport>` (or use your favorite web browser).
-   You should see the HTML output from the guestbook application.
-
+   ```console
+   $ kubectl get service guestbook
+   NAME        TYPE       CLUSTER-IP     EXTERNAL-IP   PORT(S)          AGE
+   guestbook   NodePort   10.10.10.253   <none>        3000:31208/TCP   1m
+   ```
+   
+   We can see that our `<nodeport>` is `31208`.
```

Comment:
Combine this with the next para.

We can see that our `<nodeport>` is `31200`, this means that port 3000 inside the pod is being mapped to port 31208 on the worker node. This port is randomly chosen by Kubernetes and will always be in the 31000 range.

Reply:
`will always be in the 31000 range.` is not true. It's a configurable range.